.circleci/config.yml

```diff
@@ -86,7 +86,7 @@ jobs:
             echo "Pushing release $CIRCLE_TAG..."
```

Comment:
if is using the circleci tag now need to remove this piece of code here

Reply:
Sorry, I don't understand. You commented on the "echo" command. Is that not needed? CIRCLE_TAG is indeed the correct one. Or did you mean the line below? That's still used in the docker command.